Leave/leave_interface.py

```diff
@@ -154,6 +154,23 @@ def GetReasonOfLeaves(leaves_array):
 
     return leaves_array[0]["reason"]
 
+def GetMissingMembersByRole(members_all, role, voice_channel_members):
```

Comment:
I don't think this function belongs in the leave interface. We don't currently have the most fitting place for this, so I'd put this in utils for now.

Reply:
GetMissingMemebersByRole uses functions found in *leave_interface* and I did not think it is a good idea to have *utils* import *leave_interface* when *leave_interface* already imports *utils*

Reply:
Moved the functionality to IsEveryoneHere in main.py